infohub/urls.py

```diff
@@ -1,18 +1,19 @@
-from django.conf.urls import include, patterns, url
+from django.conf.urls import include, url
 from django.contrib import admin
+from rest_framework_swagger.views import get_swagger_view
 
 admin.autodiscover()
+schema_view = get_swagger_view(title='Swagger API') 
 
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^', include('webhub.urls')),
-    url(r'^docs/', include('rest_framework_swagger.urls')),
+    url(r'^docs/', schema_view),
     url(r'^malaria/', include('malaria_web.urls', namespace='malaria')),
     url(r'^webhub/', include('webhub.urls', namespace='webhub')),
-    url(r'^signup/', include('signup.urls', namespace='signup')),
-    url(r'^profiles/', include('profiles.urls', namespace='profiles')),
+#    url(r'^signup/', include('signup.urls', namespace='signup')),
+#    url(r'^profiles/', include('profiles.urls', namespace='profiles')),
```

Comment:
This is because they were redundant. The infohub urls calls the webhub urls file which in turn contains all the profile and signup urls.